src/status.cpp

```diff
@@ -142,6 +142,7 @@ const std::map<const StatusCode, const std::string> Status::statusMessageMap = {
     {StatusCode::PIPELINE_DEMULTIPLY_ENTRY_NODE, "Demultiplication at entry node is not allowed"},
     {StatusCode::PIPELINE_TOO_LARGE_DIMENSION_SIZE_TO_DEMULTIPLY, "Too large dynamic demultiplication requested."},
     {StatusCode::PIPELINE_WRONG_DEMULTIPLEXER_GATHER_NODES_ORDER, "Demultiplexer and gather nodes are not in LIFO order"},
+    {StatusCode::PIPELINE_DEMULTIPLEXER_NO_RESULTS, "Pipeline execution aborted due to no content from a node"},
```

Comment:
As long as DL Model will satisfy demultiplexer node requirements it can be demultiplexer. I think that just log message should clarify which node is having this problem.

Reply:
DL models do not return batch 0 so @dtrawins is right - only custom nodes will return this status. I think log message should not include node name since requesting user has no idea about underlying structure of DAG. Only API resource name (pipeline name in this case).